ui/server/src/main/scala/pl/touk/nussknacker/ui/definition/EvaluatedParameterPreparer.scala

```diff
@@ -1,37 +1,37 @@
-package pl.touk.nussknacker.ui.process.uiconfig.defaults
+package pl.touk.nussknacker.ui.definition
 
+import pl.touk.nussknacker.engine.api.defaults.{NodeDefinition, ParameterDefaultValueDeterminer}
 import pl.touk.nussknacker.engine.api.definition
-import pl.touk.nussknacker.engine.definition.defaults.{NodeDefinition, ParameterDefaultValueExtractorStrategy}
 import pl.touk.nussknacker.engine.graph.evaluatedparam.Parameter
 import pl.touk.nussknacker.engine.graph.expression.Expression
 
-class ParameterEvaluatorExtractor(defaultValueEvaluator: ParameterDefaultValueExtractorStrategy) {
-  def evaluateParameters(nodeDefinition: NodeDefinition): List[Parameter] = {
-    val strategy: definition.Parameter => Option[String] = p => defaultValueEvaluator.evaluateParameterDefaultValue(nodeDefinition, p)
+class EvaluatedParameterPreparer(defaultValueEvaluator: ParameterDefaultValueDeterminer) {
+  def prepareEvaluatedParameter(nodeDefinition: NodeDefinition): List[Parameter] = {
```

Comment:
BTW prepareEvaluatedParameter and prepareEvaluatedBranchParameter could be merged into one method with some sort of Predicate passed as method param and then renamed to just `prepare`

Reply:
I suggest not to change it for now. Those branch parameters are currently not in "final" state. It should be done as a normal parameters of connections at the end. Doing such refactoring can make transition to final state even harder.

Reply:
Ok